README.rst

```diff
@@ -4,7 +4,7 @@ This extensions provides a client for the the extension **t3monitoring**.
 
 **Requirements**
 
-- TYPO3 CMS 9.5 LTS, 10.4 LTS or 11.3+
+- TYPO3 CMS 9.5 LTS, 10.4 LTS or 11.3+ and 12.4
```

Comment:
We need a dedicated branch for v9-10 now if we require 11+

Reply:
yeah, thats what I would suggest doing, as the Extension Handling changed a lot with TYPO3 v11